data/reusables/actions/jobs/section-defining-outputs-for-jobs.md

```diff
@@ -40,3 +34,49 @@ jobs:
 ```
 
 {% endraw %}
+
+{% note %}
+
+**Note:** When using strategy matrix, job outputs will be combined from all jobs inside matrix. The last executed job outputs will override matching outputs from previous jobs within same matrix.
+
+{% endnote %}
+
+{% raw %}
+
+```yaml
+jobs:
+  job1:
+    runs-on: ubuntu-latest
+    outputs:
+      output_1: ${{ steps.gen_output.outputs.output_1 }}
+      output_2: ${{ steps.gen_output.outputs.output_2 }}
+      output_3: ${{ steps.gen_output.outputs.output_3 }}
+    strategy:
+      matrix:
+        job: [1, 2, 3]
+    steps:
+      - name: Generate output
+        id: gen_output
+        run: |
+          job="${{ matrix.job }}"
+          echo "output_${job}=${job}" >> "$GITHUB_OUTPUT"
+          if [[ $job -eq 3 ]]; then
+            # slow down job so it will hopefully finish executing last
+            sleep 10
+            # override output_1 param
+            echo "output_1=${job}" >> "$GITHUB_OUTPUT"
+          fi
```

Comment:
I think this part is not needed in the example, as we would prefer for the example to be the base use case.

Reply:
Not sure what should I do here. Should I remove `steps` section completely?

Reply:
No, just remove the `if` block - I understand what you want to demonstrate, but it would be better to describe this in a warning block.

Reply:
Condition removed.